_posts/2023-05-23-GUIs-in-PyQt5.md

```diff
@@ -0,0 +1,263 @@
+# Ramin Honary
+
+  - Worked at Cross Compass since 2018
+  - Platform Team development, full stack
+  - Worked primarily on Manufacia
+  - Currently on working on ML-Platform
+
+## Why I use PyQt
+
+  - Developed some Python scripts that used **OpenCV**
+      - "Pattern Matching Kit" (will demo)
+      - "Image Cropping Kit" (in progress)
+  - OpenCV provides a API for GUI tools based on PyQt
+  - Decided to use PyQt directly instead of OpenCV specific API
```

Comment:
```suggestion
## Why I use PyQt

  - Developed some Python scripts that used **OpenCV**
      - "Pattern Matching Kit"
      - "Image Cropping Kit"
  - OpenCV provides a API for GUI tools based on PyQt
  - Decided to use PyQt directly instead of OpenCV specific API
```
I want more technical reasons why you chose PyQt but not others, if possible.
I say so because you already used some Python GUI frameworks.

Reply:
> "I want more technical reasons why you chose PyQt but not others, if possible."

I created a new slide "Other reasons to use PyQt"